app/actions/resetPasswordActions.js

```diff
@@ -0,0 +1,94 @@
+import {
+  GET_RESET_DATA_SUCCESS,
+  RESET_PASSWORD_EMAIL_SUCCESS,
+  RESET_PASSWORD_SUCCESS,
+  GET_RESET_DATA_FAILURE
+} from '../constants/constants'
+
+import { GENERAL_ERROR, UNABLE_TO_RESET, UNABLE_TO_SEND } from '../constants/messageConstants'
+
+import dmapi from '../services/apiClient'
+import { sendingRequest, setErrorMessage } from './appActions'
+
+export function handleResetPasswordSuccess() {
+  return {
+    type: RESET_PASSWORD_EMAIL_SUCCESS
+  }
+}
+
+export function sendResetPasswordEmail(values) {
+  return dispatch => {
+    dispatch(sendingRequest(true))
+    dmapi({
+      method: 'post',
+      url: '/reset-password/',
+      data: JSON.stringify(values)
+    })
+      .then(response => {
+        return response
+      })
+      .then(response => {
+        if (response.error) {
+          dispatch(setErrorMessage(UNABLE_TO_SEND))
+        } else {
+          dispatch(handleResetPasswordSuccess())
+        }
+        dispatch(sendingRequest(false))
+      })
+  }
+}
+
+export function handleGetResetDataSuccess(response) {
+  return {
+    type: GET_RESET_DATA_SUCCESS,
+    user: response.data
+  }
+}
+
+export function handleGetResetFailure() {
+  return { type: GET_RESET_DATA_FAILURE }
+}
+
+export function getUserDataFromToken(token) {
+  return dispatch => {
+    dispatch(sendingRequest(true))
+    dmapi({ url: `/reset-password/${token}` }).then(response => {
+      if (response.error) {
+        if (response.data.message) {
+          dispatch(setErrorMessage(`${response.data.message}. Try resending reset password email.`))
+        } else {
+          dispatch(setErrorMessage(GENERAL_ERROR))
+        }
+        dispatch(handleGetResetFailure())
```

Comment:
do you need this?

Reply:
It's used in the resetpasswordform to toggle pagealerts